pkg/image/validation/os.go

```diff
@@ -28,6 +28,7 @@ func validateOperatingSystem(ctx *image.Context) []FailedValidation {
 	failures = append(failures, validatePackages(&def.OperatingSystem)...)
 	failures = append(failures, validateUnattended(def)...)
 	failures = append(failures, validateRawConfig(def)...)
+	failures = append(failures, validateTimesync(def)...)
```

Comment:
It's better to pass the `OperatingSystem` object instead of the full definition.

Reply:
Good catch, will fix.